src/panels/CharmsPanel/CharmApplicationsDetails/CharmApplicationsDetails.tsx

```diff
@@ -0,0 +1,25 @@
+import { getSelectedApplications } from "store/juju/selectors";
+import { useAppSelector } from "store/store";
+
+type Props = {
+  charmURL: string;
+};
+
+const CharmApplicationsDetails = ({ charmURL }: Props): JSX.Element => {
+  const selectedApplications = useAppSelector(
+    getSelectedApplications(charmURL)
+  );
+
+  return (
+    <p className="p-form-help-text is-tick-element">
+      Apps:{" "}
+      {selectedApplications
```

Comment:
What do you think about truncating this list if there are more than say five apps? I think I suggested a tooltip, but it could also just have a "+3 more" which when clicked could append them to the existing list, and then a "show less" link could appear which would return the list to the previous state.